packages/graph/hash_graph/lib/graph/src/datastore/postgres/mod.rs

```diff
@@ -451,7 +454,20 @@ impl PostgresDatabase {
                 .change_context(InsertionError)?;
         }
 
-        // TODO: Store link references
+        // FIXME: Without `collect` we get a weird lifetime error
+        let entity_type_references = entity_type
+            .inner()
+            .link_references()
+            .into_iter()
+            .map(|(_link_uri, entity_type_uri)| entity_type_uri)
```

Comment:
Same as above, this isn't a "link" URI so to say

Reply:
See above

Reply:
I have a different answer here. This is the URI of a link _type_ it is not the URI of a link. The "links" field on the entity type is confusing, but we don't need to extend that confusion here. 

A link is an actual instance of a link type, it's data, just like an entity is different from an entity type.

I still think this should be changed.

Reply:
Fixed in e79e050e8864bfe4f92bc2209115ed70d93499dd